src/components/AddComment/index.tsx

```diff
@@ -116,6 +120,10 @@ const AddComment: VFC<AddCommentType> = ({
         if (data) {
           setText('');
           onRefresh();
+
+          if (isReply !== '') {
+            navigation.navigate('Comments', { postId, parentId: isReply });
```

Comment:
I think it's better to extract `'Comments'` screen to a constant file, import and reuse it. This way we wouldn't be confused of naming in the future.